gamemodes/terrortown/gamemode/client/cl_scoring.lua

```diff
@@ -95,19 +112,19 @@ function CLSCORE.DeclareEventDisplay(event_id, event_fns)
 	-- basic input vetting, can't check returned value types because the
 	-- functions may be impure
 	if not tonumber(event_id) then
-		Error("Event ??? display: invalid event id\n")
+		error("Event ??? display: invalid event id")
 	end
 
 	if not event_fns or not istable(event_fns) then
-		Error(Format("Event %d display: no display functions found.\n", event_id))
+		error(string.format("Event %d display: no display functions found.", event_id))
 	end
 
 	if not event_fns.text then
-		Error(Format("Event %d display: no text display function found.\n", event_id))
+		error(string.format("Event %d display: no text display function found.", event_id))
 	end
 
 	if not event_fns.icon then
-		Error(Format("Event %d display: no icon and tooltip display function found.\n", event_id))
+		error(string.format("Event %d display: no icon and tooltip display function found.", event_id))
 	end
```

Comment:
Couldn't all of these be asserts?

Reply:
I'm unfamiliar with TTT2's styling, but I avoided ``assert`` here so that string.format is only called in error cases. It saves a function call and string creation when there is no error case.